aidants_connect_habilitation/static/scss/main-habilitation.scss

```
@@ -545,6 +549,37 @@ fieldset {
   }
 }
 
+// messages -------------------------------------------
+
+.messages-list-container {
+  min-height: 250px;
+  max-height: 400px;
```

On a tout qui marche en `rem` dans la mise en page (root em = la taille de police de l'élément racine), donc plutôt ceci : 

```suggestion
  min-height: 15rem;
  max-height: 25rem;
```

---

NB. 1 rem = 16 px ici